### ISSUE TITLE
[BITV] 9.1.3.1h/3.1 - Modal dialog "File settings" - The input field for the WebDAV URL does not have a label associated with it programmatically.  (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41835]
It is recommended to implement a visual label (e.g., "WebDAV Url") using the "label" element, which would be linked to the field using the "for" and "id" attributes.

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-fa5c8d46f9-v3-n1

9.1.3.1h/11.1 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-fa5c8d46f9-v11-n1

**Comment by szaimen** [comment]
a backport to 28 seems to be missing
